src/data/fintype/basic.lean

```diff
@@ -1285,9 +1285,13 @@ fintype.of_equiv _ sym.sym_equiv_sym'.symm
   fintype.card (finset α) = 2 ^ (fintype.card α) :=
 finset.card_powerset finset.univ
 
+lemma finset.mem_powerset_len_univ_iff [fintype α] {s : finset α} {k : ℕ} :
+  s ∈ powerset_len k (univ : finset α) ↔ card s = k :=
+mem_powerset_len.trans $ and_iff_right $ subset_univ _
+
 @[simp] lemma finset.univ_filter_card_eq (α : Type*) [fintype α] (k : ℕ) :
   (finset.univ : finset (finset α)).filter (λ s, s.card = k) = finset.univ.powerset_len k :=
-by { ext, simp [finset.mem_powerset_len] }
+by { ext, simp [finset.mem_powerset_len_univ_iff] }
```

Comment:
What's the reason for this change? Does `finset.mem_powerset_len` no longer work here? I tend to think churn needs some justification.

This is the squeezed version of the old `simp`:
```suggestion
by { ext, simp only [mem_powerset_len, mem_filter, subset_univ, mem_univ] }
```

and this is the new one:
```suggestion
by { ext, simp only [mem_filter, mem_univ, true_and, mem_powerset_len_univ_iff] }
```

Reply:
I thought it would be an improvement, but I guess not.